features/cli_new.feature

```diff
@@ -71,6 +71,7 @@ Feature: Howitzer CLI New Project Creation
           Added 'config/drivers/crossbrowsertesting.rb' file
           Added 'config/drivers/headless_chrome.rb' file
           Added 'config/drivers/headless_firefox.rb' file
+          Added 'config/drivers/lambdatest.rb' file
```

Comment:
looks like it was not actualized with master. Lambda test is not a part of Selenium 4 support